.github/workflows/terraform-compatibility-matrix.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     uses: ./.github/workflows/test-suite.yml
     with:
       terraform_matrix: '["${{ matrix.terraform_version }}"]'
-      atlas_cloud_env: ${{ inputs.atlas_cloud_env }}
+      atlas_cloud_env: ${{ inputs.atlas_cloud_env || 'dev' }}
```

Comment:
what does this need to change?

Reply:
If this is scheduled on a Sunday, then the Terraform compatibility matrix tests would run against QA

Reply:
oh sorry, you're right, this calls test suite not acc test GHA